### PR TITLE
Adding support for the 'extra_disks' parameter

### PR DESCRIPTION
**Pull request by juan-acevedo-ntt**
* Adding a new parameter 'extra_disks' to add custom size data disks
* updated README.md
* updated main.tf
* updated variables.tf
* tested on TF v0.13.5

This new parameter deprecates nb_data_disk and data_disk_size_gb as it permits to add any number of data disks giving a custom size and name for each one. Example extra_disks variable:
```hcl
extra_disks = [
  {
    size = 200
    name = "logs"
  },
  {
    size = 1500
    name = "homes"
  }
]
```

### CODE REVIEW

**Comment by yupwei68** [comment]
Hi @juan-acevedo-ntt  Thanks for contribution. Would you mind add this feature a test in test/fixture and in the example in README?

**Comment by juan-acevedo-ntt** [comment]
> 
> 
> Hi @juan-acevedo-ntt Thanks for contribution. Would you mind add this feature a test in test/fixture and in the example in README?

Sure! There is already an example in the README. I will add the test.

File: README.md [hunk 1]
```diff
@@ -199,6 +199,16 @@ module "windowsservers" {
...
+  extra_disks                   = [
+                                  {
+                                    size = 50
+                                    name = "logs"
+                                  },
+                                  {
+                                    size = 200
+                                    name = "backup"
+                                  }
+                                ]
```
Comment (yupwei68) [inline README.md:211]: We could format the code in README by run "terrafmt fmt README.md". This looks a little bit strange for me. We could leave a space line and add it in the end of "windowsservers".

Comment (juan-acevedo-ntt) [reply README.md]: I ran `terraform fmt` but I do not think the formatter is doing his job properly when there is a object list as value.

File: variables.tf [hunk 2]
```diff
@@ -221,6 +221,14 @@ variable "identity_ids" {
...
 }
 
+variable "extra_disks" {
+  description = "(Optional) List of extra data disks attached to each virtual machine."
+  type = list(object({
+    name = string
+    size = number
+  }))
+  default = []
+
```
Comment (yupwei68) [inline variables.tf:231]: Obviously, the CI has failed for lack of "}".

**Review by yupwei68 — COMMENTED** [review]
Thanks for your contribution! 🚀  This PR looks almost good to me, except some minor changes.

File: variables.tf (hunk 2 above)
Comment (juan-acevedo-ntt) [reply variables.tf]: Ouch! :-(

**Review by yupwei68 — APPROVED** [review]
LGTM!

**Pull request merged**